pom.xml

```
@@ -58,8 +58,8 @@
         <legacyuiVersion>1.9.0</legacyuiVersion>
         <webservices.restVersion>2.29.0</webservices.restVersion>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <maven.compiler.source>7</maven.compiler.source>
-        <maven.compiler.target>7</maven.compiler.target>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
```

nit: Java's pretty much completely dropped the 1

```suggestion
        <maven.compiler.source>8</maven.compiler.source>
        <maven.compiler.target>8</maven.compiler.target>
```